src/selection/selection-constants.ts

```diff
@@ -25,14 +25,15 @@ export enum kwd {
   RING = 17,
   AROMATICRING = 18,
   METAL = 19,
-  NONE = 20
+  NONE = 20,
```

Comment:
I think NONE should always be last here.

Reply:
I have fixed it in latest commit. Thanks.

Reply:
Thanks @rjdkmr - all looks good I think.